lilypad/server/db/session.py

```diff
@@ -5,10 +5,17 @@
 
 from sqlmodel import Session, create_engine
 
-db_directory = os.getenv("LILYPAD_PROJECT_DIR", os.getcwd())
-engine = create_engine(
-    f"sqlite:///{db_directory}/pad.db", connect_args={"check_same_thread": False}
-)
+from ..settings import get_settings
+
+settings = get_settings()
+if settings.environment == "local" or settings.environment == "test":
+    db_directory = os.getenv("LILYPAD_PROJECT_DIR", os.getcwd())
+    engine = create_engine(
+        f"sqlite:///{db_directory}/pad.db", connect_args={"check_same_thread": False}
+    )
+else:
+    database_url = f"postgresql://{settings.db_user}:{settings.db_password}@{settings.db_host}:{settings.db_port}/{settings.db_name}"
```

Comment:
i'm assuming this means for now we are only going to accept postgres connections for non-local stuff?

Reply:
yes, unless you want another env for dialect

Reply:
i think this is fine for now we can always add more later but since we're starting with just postgres we can hold off